src/app/lib/utilities/preprocessor/rules/filterContentType.js

```diff
@@ -17,7 +18,10 @@ const filterUnknownContentTypes = data => {
       newGroup.items = group.items.filter(
         item =>
           (item.assetTypeCode || item.cpsType) &&
-          whitelist.includes(item.cpsType || item.assetTypeCode),
+          whitelist.includes(item.cpsType || item.assetTypeCode) &&
+          (item.assetTypeCode === 'PRO'
+            ? !!contentTypes.includes(item.contentType)
+            : true),
```

Comment:
this ternary can simplify to
`(item.assetTypeCode !== 'PRO' || contentTypes.includes(item.contentType))`

(Array.includes already returns a boolean, you don't need to `!!` it)

Reply:
Could we also simplify the mapping of `group` and filtering of `group.items`. Its getting hard to follow.

Reply:
Done!